install/0000_90_machine-api-operator_04_alertrules.yaml

```diff
@@ -4,11 +4,11 @@ metadata:
   labels:
     prometheus: k8s
     role: alert-rules
-  name: machineapioperator-rules
+  name: prometheus-prow-rules
```

Comment:
Maybe `machine-api-operator-prometheus-rules`? Why is it called `prometheus-prow-rules`?